### DISCUSSION TITLE
Skill assessment for coupled biological/physical models of marine systems

### DISCUSSION THREAD

**Discussion by ecomodeller** [DHI/modelskill#16] [Examples from literature]
[Stow et al, 2009](https://www.sciencedirect.com/science/article/pii/S0924796308001103)
## Abstract
Coupled biological/physical models of marine systems serve many purposes including the synthesis of information, hypothesis generation, and as a tool for numerical experimentation. However, marine system models are increasingly used for prediction to support high-stakes decision-making. In such applications it is imperative that a rigorous model skill assessment is conducted so that the model's capabilities are tested and understood. Herein, **we review several metrics** and approaches useful to evaluate model skill. The definition of skill and the determination of the skill level necessary for a given application is context specific and **no single metric is likely to reveal all aspects of model skill**. Thus, **we recommend the use of several metrics**, in concert, to provide a more thorough appraisal. The routine application and presentation of rigorous skill assessment metrics will also serve the broader interests of the modeling community, ultimately resulting in improved forecasting abilities as well as helping us recognize our limitations.

**Comment by jsmariegaard** [comment]
Which metrics are they recommending?

**Reply by ecomodeller** [reply]
They suggest these six metrics. But also notes that they are all sensitive to phase errors.
![image](https://user-images.githubusercontent.com/614215/107742777-f9558300-6d0f-11eb-8440-bf96e2a06313.png)
